src/js/model/rules/scales.js

```diff
@@ -99,7 +99,7 @@ module.exports = function(parsed) {
     if (!def) {
       continue;
     }
-    if (!curr || !equals(def, curr)) {
+    if (!curr || !equals.call(this, def, curr)) {
```

Comment:
This was our sneaky sometimes-error for dropping fields: `.call` was used in all other cases, just not here. I'd recommend down the line to make this a function that takes its context as an argument to avoid excessive binding, `equals(this, def, curr)` e.g.